.dockerignore

```diff
@@ -0,0 +1,5 @@
+.cfme_venv/
+build/
+dist/
+docs/_build
+*.pyc
```

Comment:
Some more dirs to add here:
.cache/
.pytest_cache/
.idea/ (for those on pycharm :P)

And not sure what we want to do about .yaml_key -- we could copy it in as long as we're not releasing the built image with the yaml key publicly